### DISCUSSION TITLE
Getting dosemu2 to resize in tmux

### DISCUSSION THREAD

**Discussion by dialognewmedia-mws** [dosemu2/dosemu2#1790] [General]
Inside my .tmux.conf, the following two lines allow regular TUI programs to be resized to the currently active terminal

set -g window-size latest
set -g aggressive-resize on

Alas, when using dosemu2 terminal mode, instead of expanding, the emulated DOS screen simply fills the extra space with black.

FreeDOS Edit, not expanded on a larger active terminal:
![dosemu2-tmux](https://user-images.githubusercontent.com/3652947/191430616-8841b60c-177a-4fa7-a628-8d02714c08d9.png)

Tilde editor, properly expanded:
![tilde](https://user-images.githubusercontent.com/3652947/191431186-3ab310f8-2754-4f5c-9df8-dbdae425c76b.png)

**Comment by stsp** [comment]
This is not a default behavior.
What exactly dosemu2 version
do you run, and have you set
perhaps `$_fixed_term_size`
to some value?

**Reply by dialognewmedia-mws** [reply]
$: dosemu --version
dosemu2-2.0pre9
Revision: 7076

Here is my entire tmux conf:

set -g status off
set-option -g mouse on
set-option -g default-terminal "xterm"
set -g window-size latest
set -g aggressive-resize on

**Reply by stsp** [reply]
Version is likely inaccurate.
If you use pre-packaged build,
you need to query version with
the package manager.

tmux.conf is not interesting, I
was asking about a dosemu2
option.

Anyway, the problem is that
resize is impossible.
You need to disable resize in
tmux and start dosemu2 when
the window is already large.
DOS programs do not have
any kind of "resize notifications",
so the only way to get the large
screen is to enlarge the screen
before starting dosemu2.
We can add the resizing capability
to xmode.com, in which case you
will be able to exit your edit.exe,
resize the window with the command
and start edit.exe again.
